dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/fat/src/io/openliberty/webcontainer/servlet60/fat/FATSuite.java

```diff
@@ -50,6 +54,13 @@
 })
 public class FATSuite {
 
+    // EE11 requires Java 21
+    // If we only specify EE11 for lite mode it will cause no tests to run which causes an error.
+    // If we are running on Java 11 or 17 have EE10 be the lite mode test to run.
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests.with(new EmptyAction().conditionalFullFATOnly(EmptyAction.GREATER_THAN_OR_EQUAL_JAVA_21))
+                    .andWith(FeatureReplacementAction.EE11_FEATURES());
+
```

Comment:
What happens when FULL mode is run for Java 11 or 17?

Reply:
@volosied only the EE10 (Servlet 6.0) is executed and the following is put in the logs for instance with java 17: 

`Skipping action 'JakartaEE11 FAT repeat action' because the java level is too low.`

`JavaInfo                       <init>                         I major=17,  minor=0, service release=0, fixpack=0,  vendor=IBM,  javaHome=<path-to-jdk>\jdk-17.0.7+7, criuSupported=Optional.empty`